cg/store/models.py

```diff
@@ -1048,6 +1048,13 @@ class InstrumentRun(Base):
         "polymorphic_on": "type",
     }
 
+    def to_dict(self, samples: bool = False):
+        """Represent as dictionary"""
+        data = to_dict(model_instance=InstrumentRun)
+        if samples:
+            data["samples"] = [sample.to_dict() for sample in self.device.samples]
+        return data
```

Comment:
I think if we do not use the to_dict anywhere now, we should avoid doing it in the future. Instead we should start using pydantic models for data transfers.